reconcile/utils/saasherder/saasherder.py

```diff
@@ -1267,8 +1272,11 @@ def populate_desired_state_saas_file(
                 github=spec.github,
                 target_config_hash=spec.target_config_hash,
             )
-        except Exception:
-            # log message send in _process_template
+        except Exception as e:
+            # error log message send in _process_template. We log here debug to have a
+            # safeguard in case something breaks there unexpectedly. We cannot just
+            # register an error without logging as inventory errors don't have details.
+            logging.debug(f"Error in populate_desired_state_saas_file: {e}")
```

Comment:
If `re.register_error` can collect actual error object and print details somewhere, that will be best, but also requires bigger change. The exception is eaten here, won't appear  in APM or glitchtip.

Reply:
yeah, the way things are managed are not ideal.